src/terminal.c

```diff
@@ -75,6 +72,19 @@ __RCSID("$NetBSD: terminal.c,v 1.32 2016/05/09 21:46:56 christos Exp $");
 #include "el.h"
 #include "fcns.h"
 
+#include <dlfcn.h>
+
+void *dl_handle = NULL;
+char *(*tgoto) (const char *cap, int col, int row);
+int (*tgetent) (char *bp, const char *name);
+int (*tgetflag) (char *id);
+int (*tgetnum) (char *id);
+char *(*tgetstr) (char *id, char **area);
+int (*tputs) (const char *str, int affcnt, int (*putc)(int));
+
+#define	NCURSES_HIGH_VERSION	10
```

Comment:
How about changing the prefix name from NCURSES to TINFOLIB ?

Reply:
It looks better.
Thank you!